### PR TITLE
refactor to use es6 classes

### PR DESCRIPTION
**Pull request by tbo47**
the build is broken because I made a mistake while refactoring `src/og/webgl/Handler.js`
this PR fixes it
@Zemledelec @pavletto

### CODE REVIEW

**Review by Zemledelec — APPROVED** [review]

**Comment by tbo47** [comment]
@Zemledelec  I don't have write access. I can not merge this PR.

**Pull request merged**
